app/src/Pages/ResetPassword/ResetPassword.js

```diff
@@ -15,13 +14,14 @@ class ResetPassword extends React.Component {
   }
 
   componentDidUpdate(){
-    const { error, flashMessage, history, isLoading } = this.props;
+    const { error, flashMessage, history, isLoading, logout } = this.props;
     if (this.state.submitted && !isLoading){
     if(error){
       flashMessage("Password update failed", {props:{variant:'error'}});
     }else {
+      logout();
       flashMessage("Password updated", {props:{variant:'success'}});
-      history.push('/dashboard');
+      history.push('/sign-in');
```

Comment:
I think this is fine for now. It might be better if we move these into the thunk side effects.

I.e. in the logout action, add an action for history.push('/sign-in'). Keeps the component clean, moves all business logic to actions.

Reply:
Sounds good, refactored history and moved forwards and flash to ducks.